migrations/1_initial_migration.js

```diff
@@ -2,11 +2,15 @@ const path = require('path')
 const fs = require('fs').promises
 const WorkflowCompile = require('@truffle/workflow-compile')
 
-const WEI_TO_DEPOSIT = web3.utils.toWei("10", "ether")
-const DAI_TO_CREATE  = web3.utils.toWei("80", "ether")
+const ETH_TO_DEPOSIT = 10n
+const DAI_TO_CREATE  = 80n
```

Comment:
We should consider using `number` as a double for these rather than `bigint` as an integer.  This would allow us to enter `80.001` and as long as it was within 52-bits of precision, it will be exact (which will be the case in any situation where we are typing numbers into code by hand).  Helper methods like `toDai(attodai: bigint): number` and `toAttodai(dai: number): bigint` can be trivially created to facilitate conversion between the two.  Internally, we should use `attodai` as a `bigint` everywhere, but make it easy to pass in/around `dai` and have constants that are in `dai` (like here).